index.bs

```diff
@@ -1314,6 +1315,15 @@ The dictionary SHALL have the following attributes:
    [Section 5.3](https://www.rfc-editor.org/rfc/rfc9002#section-5.3).
 : <dfn for="WebTransportStats" dict-member>minRtt</dfn>
 :: The minimum round-trip time observed on the entire connection.
+: <dfn for="WebTransportStats" dict-member>estimatedSendRate</dfn>
+:: The estimated rate at which queued data will be sent by the user agent, in bits per second.
+   This rate applies to all streams and datagrams that share a [=WebTransport session=]
+   and is calculated by the congestion control algorithm (potentially chosen by
+   {{WebTransport/congestionControl}}). If this [=WebTransport session=] is
+   pooled with others in a shared [=connection=], the value is undefined.
```

Comment:
I gather this is to satisfy @martinthomson's concern in https://github.com/w3c/webtransport/pull/421#discussion_r1107885111?

This might limit media ingest to non-pooled connections. Are we OK with that? Is there no strategy a user agent could employ instead (e.g. divide 50/50 and omit other traffic from the result)?

Or perhaps this is fine?

Reply:
To start with, this speaks to the feature detection angle more strongly.

The primary concern here is with cross-origin WebTransport sessions.  That is, when example.com establishes a session toward https://wt.example/upload and wants to know stuff about that connection.  If that connection is used to load stuff for other purposes that wt.example might have (within the same top-level browsing context), then information about size and timing might leak through.  We might assume that example.com is not already in a position to observe that much and so adding this new information would be irresponsible.

After getting more comfortable with my own understanding of this situation, I believe that it is already the case that sites that can make queries of cross-site content can learn stuff about how the connection is being used.  So we're just changing the fidelity of the measurements and maybe changing/constraining how the browser might be able to act to protect that information from observation.  For instance, in the current regime, a browser could isolate requests from different sources in time, so that requests never contend for capacity, at the cost of timing contention; WebTransport might make that sort of trick harder to pull off.

I'm almost at the point where I would instead say that this requirement can be removed, though perhaps we can do that in a follow-up.